packages/likes/src/likes.tsx

```diff
@@ -59,6 +59,16 @@ function Likes({
     { type: 'no', label: noLabel, icon: 'ri-thumb-down-line' },
   ];
 
+  useEffect(() => {
+    let pending = session.get('osc-resource-vote-pending');
+    if (pending && pending[resource.id]) {
+      if (currentUser && currentUser.role) {
+        doVote(null, pending[resource.id])
```

Comment:
Waarom wordt de doVote methode met een null aangeroepen?
Overschrijft dit niet dan de pending yes of no vote?

Reply:
Nee, zie regel 71:
```
async function doVote(e, value) {
```